### ISSUE TITLE
Weird sublinks/sitelinks on Google

### ISSUE THREAD

**Issue by pranasziaukas** [balena-io/docs#1622]
Not sure where to post it but some sublinks/sitelinks on Google pointing to balena are weird

<img width="865" alt="Screenshot 2021-03-16 at 17 21 41" src="https://user-images.githubusercontent.com/26874494/111493049-e2a6af80-8745-11eb-9750-11914161e57c.png">

This concerns https://github.com/balena-io/balena-cli/blob/master/doc/cli.markdown

Thread [here](https://www.flowdock.com/app/rulemotion/i-docs/threads/SKvwPo46Y5mCbo7VGsExKkPk9kV), cc @vipulgupta2048

**Comment by pdcastro** [comment]
To confirm, the problem / complaint is that Google is rendering `<` as `&lt;`, is this right? If so, this looks like a bug in the Google search rendering, something to report to Google. :-) But we may be able to work around it on our side, by avoiding the `<` and `>` characters in markdown section titles.

> This concerns https://github.com/balena-io/balena-cli/blob/master/doc/cli.markdown

Yes, which is [auto generated from Typescript source code](https://github.com/balena-io/balena-cli/blob/master/CONTRIBUTING.md#editing-documentation-files-readme-install-reference-website). @srlowe, do the `<` and `>` come from this patch file? -
https://github.com/balena-io/balena-cli/blob/v12.41.3/patches/all/%40oclif%2Bplugin-help%2B3.2.0.patch#L21

**Comment by pranasziaukas** [comment]
> To confirm, the problem / complaint is that Google is rendering `<` as `&lt;`, is this right? If so, this looks like a bug in the Google search rendering, something to report to Google.

Right. I found [this](https://support.google.com/adspolicy/answer/1054210?hl=en) article about sitelink ads but I'm speculating that it might apply to sitelinks in general
> The following is not allowed: Punctuation and symbols in the link text or description that serve no purpose other than to draw attention to the ad
Examples: Exclamation marks, punctuation at the beginning of the text, adding a ► symbol

So that could be not a bug but a feature to discourage the `>>>CLICK ME!!!<<<` types of sitelinks.

Once again, I know very little about SEO and how Google indexes its content so I just reported a quirk I found as a user.
